build-npm/action.yml

```diff
@@ -0,0 +1,87 @@
+name: Build NPM
+descripion: Build a npm-module
+
+inputs:
+  github_artifact_retention:
+    description: 'Time during which artifact are kept in Github infrastructure'
+    default: '2'
+  tests_module_path:
```

Comment:
tests_module_path is not used in the action, it can probably be removed until we decide what to do with NPM test modules